### PR TITLE
Add RPM linux build target

### LINKED ISSUE #230: [SUGGESTION] - Add rpm packages to new releases
Without a Flatpak release available there is no way to install Bruno in RPM based systems like Fedora or OpenSUSE without building the entire project each update. I suggest releasing .rpm packages with the upcoming releases.

### PR DESCRIPTION
**Pull request by brunohpaiva**
Adds RPM as a Linux target to the electron-builder config.
The output is a `bruno_%version%_x86_64_linux.rpm` file in the `packages/bruno-electron/out` directory.
Resolves #230.

### CODE REVIEW

File: packages/bruno-electron/electron-builder-config.js
Comment (brunohpaiva) [inline packages/bruno-electron/electron-builder-config.js:1]: The whole file was changed as a result of the pre-commit hook "husky". The only change here was actually the addition of `'rpm'` to the `linux.target` array.

**Comment by brunohpaiva** [comment]
Running on Fedora 38, installed via the RPM file

image

**Comment by helloanoop** [comment]
Thanks @brunohpaiva  :) !

**Comment by rombert** [comment]
I see that https://github.com/usebruno/bruno/releases/tag/v0.24.0 was released yesterday but the RPM files are not present. Not a big deal as I can still use the AppImage file, but I assume the plan is to create RPMs for releases as well, right?

**Comment by helloanoop** [comment]
Yes @rombert Will add support for downloading rpm files and eventually support for `yum install bruno`

**Comment by manojbaishya** [comment]
I had no issues with a local build on Fedora 38 from the main branch:

```bash
git clone https://github.com/usebruno/bruno.git
cd bruno
npm install
npm run build:electron:rpm
cd packages/bruno-electron/out
dnf install bruno_0.24.0_x86_64_linux.rpm
```

**Pull request merged**